src/main/java/seedu/quotesify/commands/EditCommand.java

```diff
@@ -44,13 +49,34 @@ public void execute(TextUi ui) {
             CategoryList categoryList = (CategoryList) ListManager.getList(ListManager.CATEGORY_LIST);
             editCategory(categoryList, ui);
             break;
+        case TAG_QUOTE:
+            QuoteList quotes = (QuoteList) ListManager.getList(ListManager.QUOTE_LIST);
+            editQuote(quotes, ui);
+            break;
         default:
             ui.printListOfEditCommands();
             break;
         }
         Storage.save();
```

Comment:
By right execute() should take in Storage as parameter and this line of code should be storage.save() instead due to the previous commit I pushed just now. Not sure why Git did not flag this as a merge conflict though.

Can you try to pull the latest one and see if the code updates?

Reply:
Got it updated to the PR!